dex/src/main/scala/com/wavesplatform/dex/settings/LpAccountsSettings.scala

```diff
@@ -0,0 +1,27 @@
+package com.wavesplatform.dex.settings
+
+import com.wavesplatform.crypto.base.Base58
+import com.wavesplatform.dex.domain.account.{Address, PublicKey}
+
+import java.io._
+import scala.jdk.CollectionConverters._
+
+case class LpAccountsSettings(filePath: String) {
+
+  lazy val publicKeys: Set[PublicKey] = {
```

Comment:
i think we need lazy val only for ```addresses``` (for chain id proper handling). ```publicKeys``` have to be initialized immediately in order to prevent matcher bootstrap earlier whether it needs to.